### DISCUSSION TITLE
Is there somewhere a list of parameters we can play with?

### DISCUSSION THREAD

**Discussion by WiliTest** [Windos/BurntToast#149] [Q&A]
I did look for this list in the `README.md` but I did not find it (it would be great  to have it there).
 
I would like to set a delay before killing the notification, but I did not find more parameters that the one shown in the examples.

**Comment by Windos** [comment]
You can list out all parameters via the built-in help, `(help New-BurntToastNotification -Detailed).Parameters`

In general, the help across the board needs work. I'm working on a documentation website which will provide a lot more context on everything, but it's a long way off.

I'm not 100% on what you're trying to achieve, there are a couple of levers you can pull to affect the duration of the toasts.

To make it display for a longer time, change the "Duration" (n.b. this is done via the `New-BTContent` function, as shown [in this example](https://github.com/Windos/Talks/blob/main/2020/10%20-%20October/14%20-%20SoCalPSUG%20-%20Croque%20Monsieur%20-%20Step%20up%20Your%20Toast%20Notification%20Game/Demos/2_Duration.ps1).)

It can be made even longer by changing the scenario (again, done via `New-BTContent`, here's [an example](https://github.com/Windos/Talks/blob/main/2020/10%20-%20October/14%20-%20SoCalPSUG%20-%20Croque%20Monsieur%20-%20Step%20up%20Your%20Toast%20Notification%20Game/Demos/3_Scenario.ps1).)

`New-BurntToastNotification` does have an ExpirationTime parameter which takes a datetime, but this doesn't extend the time for the toast. Setting this removes the toast from the action center (and the screen if it happens to still be up), but I've noticed that the minimum time is 3 seconds:

```powershell
New-BurntToastNotification -ExpirationTime (Get-Date).AddSeconds(3)
```

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
